### DISCUSSION TITLE
1.0.1

### DISCUSSION THREAD

**Discussion by Ganavi-DevTools** [Ganavi-DevTools/Ganavi-test#2] [General]
# this is the updated version

<hr /><em>This discussion was created from the release <a href='https://github.com/Ganavi-DevTools/Ganavi-test/releases/tag/tag'>1.0.1</a>.</em>

**Comment by Ganavi-DevTools** [comment]
### hey!! happy about the new release

**Comment by Deepika-DevTools** [comment]
Unlocked
